src/singa.py

```diff
@@ -0,0 +1,15 @@
+import os
```

Comment:
I think this file should be inside tool/python/.
it is used 
  
    ./bin/singa-run.sh -exec tool/python/singa.py -conf ...